packages/language-server/src/xml-view-diagnostics.ts

```diff
@@ -56,6 +56,10 @@ function validationIssuesToLspDiagnostics(
         return {
           ...commonDiagnosticPros,
         };
+      case "CardinalityOfAggregation":
```

Comment:
you can simply add this to the list of cases above which "fallthrough" for the base behavior.